app/helpers/api-util.js

```diff
@@ -167,26 +167,27 @@ export async function getByIdPassport(id) {
 
 // PROVERBS
 
-export async function getAllProverbs() {
+export async function getProverbs() {
   try {
     const client = await clientPromise;
     const db = client.db("roads");
 
-    const allProverbs = await db.collection("proverbs").find({}).toArray();
+    const currentDate = new Date();
+    let currentMonth = currentDate.getMonth() + 1;
```

Comment:
const currentMonth = new Date().getMonth() + 1;

Reply:
well noted, thank you so much!